Build error on Debian 11

Currently the oldest distribution OpenSCAD supports is Debian 11. After upgrading to Manifold v3.2.1 via https://github.com/openscad/openscad/pull/6024 the [OBS builds](https://build.opensuse.org/package/show/home:t-paul/OpenSCAD) fail with a compilation error ([logs](https://build.opensuse.org/package/live_build_log/home:t-paul/OpenSCAD/Debian_11/x86_64)).

```
[  135s] /usr/src/packages/BUILD/submodules/manifold/src/impl.cpp:152:49: error: 'tbb::interface7::task_arena::priority' has not been declared
[  135s]   152 | tbb::task_arena gc_arena(1, 1, tbb::task_arena::priority::low);
[  135s]       |                                                 ^~~~~~~~
[  135s] cc1plus: note: unrecognized command-line option '-Wno-shorten-64-to-32' may have been intended to silence earlier diagnostics
[  135s] cc1plus: note: unrecognized command-line option '-Wno-unknown-warning-option' may have been intended to silence earlier diagnostics
[  135s] make[3]: *** [submodules/manifold/src/CMakeFiles/manifold.dir/build.make:163: submodules/manifold/src/CMakeFiles/manifold.dir/impl.cpp.o] Error 1
```

With Debian 13 around the corner, there's maybe not a huge need to continue supporting Debian 11, but if possible it would be nice to keep it around for a bit longer.

Build env:
- gcc-10-10.2.1-6
- cmake-3.18.4-2+deb11u1
- libtbb-dev-2020.3-1

--- a/src/vec.h
+++ b/src/vec.h
@@ -232,11 +232,7 @@ class Vec : public VecView<T> {
 #if !__has_feature(address_sanitizer)
 #if (MANIFOLD_PAR == 1)
     if (size * sizeof(T) > ASYNC_FREE_THRESHOLD)
-#if (TBB_VERSION_MAJOR < 2021)
-      gc_arena.enqueue([ptr]() { free(ptr); }, tbb::priority_t::priority_low);
-#else
       gc_arena.enqueue([ptr]() { free(ptr); });
-#endif
     else
 #endif
 #endif

